### PR TITLE
ROX-14987: Fix wrong info for anomalous external flows count

### PR DESCRIPTION
**Pull request by sachaudh**
## Description

This PR fixes an issue with the deployment details tab showing the wrong info for the number of external flows for this deployment

Screenshot 2023-02-08 at 10 30 06 AM

## Checklist
- [ ] Investigated and inspected CI test results
- [x] Unit test and regression tests added
- ~[ ] Evaluated and added CHANGELOG entry if required~
- ~[ ] Determined and documented upgrade steps~
- ~[ ] Documented user facing changes (create PR based on [openshift/openshift-docs]~(https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))~

### CODE REVIEW

File: ui/apps/platform/src/Containers/NetworkGraph/utils/networkGraphUtils.test.ts [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+                            protocol: 'L4_PROTOCOL_TCP',
+                            lastActiveTimestamp: '2023-02-08T18:24:50.470158762Z',
+                        },
+                        {
+                            port: 80,
+                            protocol: 'L4_PROTOCOL_TCP',
+                            lastActiveTimestamp: '2023-02-08T18:24:50.470158762Z',
+                        },
+                    ],
+                    targetToSourceProperties: [
```
Comment (sachaudh) [inline ui/apps/platform/src/Containers/NetworkGraph/utils/networkGraphUtils.test.ts:85]: Considering that this field is only present when the `isBidirectional` value is true, maybe we can tighten up our typing for the `CustomEdgeModel` to reflect that. Right now we just have `targetToSourceProperties` as an optional field

Comment (alwayshooin) [reply ui/apps/platform/src/Containers/NetworkGraph/utils/networkGraphUtils.test.ts:85]: i think that makes sense, maybe have two different edge types to accommodate for `DirectionalEdge` and `BidirectionalEdge`

File: ui/apps/platform/src/Containers/NetworkGraph/utils/networkGraphUtils.test.ts [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+                    tag: '4',
+                },
+            };
+
+            const numFlows = getNumFlowsFromEdge(edge);
+
+            expect(numFlows).toEqual(4);
+        });
+    });
+});
```
Comment (alwayshooin) [inline ui/apps/platform/src/Containers/NetworkGraph/utils/networkGraphUtils.test.ts:106]: tests 🥲

**Review by alwayshooin — APPROVED** [review]

**Pull request merged**